### ISSUE TITLE
Adopt different game resolutions in different languages

### ISSUE THREAD

**Issue by DavidLiang1129** [Tuxemon/Tuxemon#1475]
**Is your feature request related to a problem? Please describe.**
I came up with this idea because Tuxemon's font for displaying ideograms is too small. Inspired by question #1474, I thought we could change the font size in a disguised way by changing the resolution (of course, there might be a better way).
————————————————————
Cheese:
我想出这个主意是因为Tuxemon用来显示表意文字的字体太小了。受到问题#1474的启发，我想我们可以通过改变分辨率来变相地改变字体大小(当然，可能有更好的方法)。
**Describe the solution you'd like**
I wanted Tuxemon to have different font sizes for different locales.
————————————————————
Chinese:
我希望Tuxemon在不同的语言环境中拥有不同的字体大小。
**Describe alternatives you've considered**
In fact, this itself is a convenient alternative to changing the font size.
————————————————————
Chinese:
事实上，这本身就是更改字体大小的一种方便的替代方法。
**Additional context**
In fact, some games display ideograms in fonts larger than Latin letters, and I'll use Pokemon FireRed as an example
The languages in the screenshot below are English, Japanese and Chinese
————————————————————
Chinese:
事实上，有些游戏以比拉丁字母更大的字体显示表意文字，我将以宝可梦火红版为例
下面截图中的语言是英语、日语和中文
![口袋妖怪（精灵宝可梦）火红 美版v1.1_1670803926040.png](https://user-images.githubusercontent.com/118408976/206937908-b643c5c9-28e9-41ac-90fa-f8fd26e34257.png)![口袋妖怪（精灵宝可梦）火红 日版_1670803970219.png](https://user-images.githubusercontent.com/118408976/206937920-7ea87133-1733-425d-ad41-0e4423c057e8.png)![精灵宝可梦火红2012中文版[口袋群星SP汉化组]_1670804026718.png](https://user-images.githubusercontent.com/118408976/206937940-3845d92d-b67d-47b7-a991-304a20c737f5.png)![口袋妖怪（精灵宝可梦）火红 美版v1.1_1670804051294.png](https://user-images.githubusercontent.com/118408976/206937957-342cfd30-40b7-4590-a44b-d6c6caca6ce9.png)![口袋妖怪（精灵宝可梦）火红 日版_1670804062323.png](https://user-images.githubusercontent.com/118408976/206937965-be5c8943-d3e4-47b1-872e-e65a7b99fe76.png)![精灵宝可梦火红2012中文版[口袋群星SP汉化组]_1670804071001.png](https://user-images.githubusercontent.com/118408976/206937983-58dc546c-0e6d-4b0c-b03a-027195ec4b80.png)![口袋妖怪（精灵宝可梦）火红 美版v1.1_1670804253166.png](https://user-images.githubusercontent.com/118408976/206938000-71c92588-6cd0-48a3-98c7-21cee5eb8cd9.png)![口袋妖怪（精灵宝可梦）火红 日版_1670804301936.png](https://user-images.githubusercontent.com/118408976/206938009-e5dec02b-074c-42d4-99ae-0c5d999b37b5.png)![精灵宝可梦火红2012中文版[口袋群星SP汉化组]_1670804370362.png](https://user-images.githubusercontent.com/118408976/206938021-b01e2887-3717-40b6-87c2-a90f8da00c15.png)

**Comment by DavidLiang1129** [comment]
But the problem is in no hurry to be solved.

**Comment by NeonFlames** [comment]
Is this still a problem?
If so I can take a look and see what I can do.

**Comment by DavidLiang1129** [comment]
> Is this still a problem?
> If so I can take a look and see what I can do.

Yes, it's still a problem thanks (I don't think it would be a problem if it was Tuxemon on a computer, but if it was on a cell phone or handheld game, the font would be too small)

**Comment by freshreplicant** [comment]
Possibly related: 

#242 
#470
